docs/topics/dynamic-content.rst

```diff
@@ -246,6 +258,7 @@ along with `scrapy-selenium`_ for seamless integration.
 .. _headless browser: https://en.wikipedia.org/wiki/Headless_browser
 .. _JavaScript: https://en.wikipedia.org/wiki/JavaScript
 .. _js2xml: https://github.com/scrapinghub/js2xml
+.. _chompjs: https://github.com/Nykakin/chompjs
```

Comment:
:lipstick: It should remain in alphabetical order.

Reply:
Okay